plugins/meta/flannel/flannel.go

```diff
@@ -182,7 +206,7 @@ func cmdAdd(args *skel.CmdArgs) error {
 
 	if !hasKey(n.Delegate, "ipMasq") {
 		// if flannel is not doing ipmasq, we should
-		ipmasq := !fenv.ipmasq
+		ipmasq := !*fenv.ipmasq
```

Comment:
I got that, I just didn't notice you change this line in the original
commit.

On Wed, Oct 7, 2015 at 6:03 PM, Eugene Yakubovich <notifications@github.com>
wrote:

> In plugins/meta/flannel/flannel.go
> <https://github.com/appc/cni/pull/73#discussion_r41466924>:
>
> > @@ -182,7 +206,7 @@ func cmdAdd(args *skel.CmdArgs) error {
> >
> >  	if !hasKey(n.Delegate, "ipMasq") {
> >  		// if flannel is not doing ipmasq, we should
> > -		ipmasq := !fenv.ipmasq
> > +		ipmasq := !*fenv.ipmasq
>
> I made all fields pointers so I can use nil as not set.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/appc/cni/pull/73/files#r41466924>.
>